### DISCUSSION TITLE
Bedrock cachePoint

### DISCUSSION THREAD

**Discussion by jarrodlugo** [aws/aws-sdk-java#3168] [Q&A]
How can I set a `cachePoint` via the Bedrock API?  I don't see it as an option on `ContentBlock` in 2.29.34 of the Java SDK, and I cannot find another way to set it when sending a `Message`.

RE: [https://docs.aws.amazon.com/bedrock/latest/userguide/prompt-caching.html](https://docs.aws.amazon.com/bedrock/latest/userguide/prompt-caching.html)

Thanks!

**Comment by debora-ito** [comment]
My guess is that because prompt caching is still in preview mode, it's not available via SDKs yet. I've reached out to the Bedrock team to confirm.

Have you [signed up](https://aws.amazon.com/bedrock/prompt-caching/) to participate in the preview?

**Comment by jarrodlugo** [comment]
Thanks Debora!  I just submitted a request to be added to the preview.  But yeah - we'd still need a way to set that flag on certain ContentBlocks/etc via the Java SDK.
